lms/templates/courseware/info.html

```diff
@@ -54,7 +54,7 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
 <section class="container">
   <div class="home">
     <div class="page-header-main">
-        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.id.org, course_name=course.id.course) | h}</h1>
+        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default) | h}</h1>
         <h2 class="page-subtitle">${course.display_name | h}</h2>
```

Comment:
Thanks @robrap I failed to look beyond the page-title, so missed `course.display_name`. I'll switch that to maintain consistent style conventions, though.

So we won't need to worry about the `| h` filter since this string will be displayed as plain text. I'm worried about changing the rest of the mako template in this PR. The scope of this one is to display the welcome header with the `display_number` course attribute.